broker/src/main/java/io/camunda/zeebe/broker/bootstrap/DiskSpaceUsageMonitorStep.java

```diff
@@ -23,14 +27,43 @@ void startupInternal(
       final ActorFuture<BrokerStartupContext> startupFuture) {
 
     final var data = brokerStartupContext.getBrokerConfiguration().getData();
+
+    if (!data.isDiskUsageMonitoringEnabled()) {
+      brokerStartupContext.setDiskSpaceUsageMonitor(new DisabledDiskUsageMonitor());
+      startupFuture.complete(brokerStartupContext);
+      return;
+    }
+
+    startDiskUsageMonitorActor(brokerStartupContext, startupFuture, data);
+  }
+
+  @Override
+  void shutdownInternal(
+      final BrokerStartupContext brokerShutdownContext,
+      final ConcurrencyControl concurrencyControl,
+      final ActorFuture<BrokerStartupContext> shutdownFuture) {
+
+    final var diskSpaceUsageMonitor = brokerShutdownContext.getDiskSpaceUsageMonitor();
+    if (diskSpaceUsageMonitor instanceof DiskSpaceUsageMonitorActor actor) {
```

Comment:
:wrench: It would be better to add a `close` method to the `DiskSpaceUsageMonitorInterface` interface.

Reply:
Yeah, good point :+1: I've just added an `extends AsyncClosable` to the `DiskUsageMonitor` interface to implement this.